stl/inc/yvals_core.h

```diff
@@ -1124,11 +1124,7 @@
 #define __cpp_lib_bind_front        201907L
 #define __cpp_lib_bit_cast          201806L
 
-#ifdef __clang__ // TRANSITION, VSO-1020212
-// a future MSVC update will embed CPU feature detection into <bit> intrinsics
-#define __cpp_lib_bitops 201907L
-#endif // __clang__
-
+#define __cpp_lib_bitops               201907L
 #define __cpp_lib_bounded_array_traits 201902L
 

```

Comment:
Below, `__cpp_lib_int_pow2` should be unconditionally defined (in C++20 mode).

Reply:
This seems to be still unresolved.